spec/unit/pops/parser/pn_parser_spec.rb

```diff
@@ -0,0 +1,103 @@
+#! /usr/bin/env ruby
+require 'spec_helper'
+require 'puppet/pops'
+require 'puppet/pops/pn'
+
+module Puppet::Pops
+module Parser
+
+describe 'Puppet::Pops::Parser::PNParser' do
+  context 'when parsing literals' do
+    it 'can parse boolean true' do
```

Comment:
\<NitPick>\
Not going to have you rephrase all of the tests, but they should really be written as examples - as they are written now, they really communicate nothing except that the parser does not crash for the given input... It could have said "parses 'true' to boolean true". Almost all labels have this kind of issue. (For this section, you could have used the prefix "when parsing literals", and continued with something like "'true' results in boolean true". Further down (more complex input) a bit more is needed as explanation (cannot just use the string form of what is being parsed) e.g. "can parse a map", could be "a PN map on the form {:key val,...} results in a PN:Map representation"\
\</NitPick>

I have other small issues with the example texts as they stand, I am pointing out some of them in separate comments.

Reply:
Should say 'can parse boolean "true"' (missing quotes around true).